lib/resources/package.rb

```diff
@@ -20,7 +20,9 @@ class Package < Inspec.resource(1)
       end
     "
 
-    def initialize(package_name = nil, opts = {}) # rubocop:disable Metrics/AbcSize
+    def initialize(package_name = nil, opts = {}) # rubocop:disable Metrics/AbcSize, Metrics/PerceivedComplexity, Metrics/CyclomaticComplexity
```

Comment:
Maybe we should change this to:

```ruby
def initialize(package_name, opts = {})
```

... such that Ruby enforces that `package_name` must be specified? While I agree that the error message isn't as clear as the one you've provided, we should be consistent with other resources that have required parameters. 

We can always add some arity magic in the future with better error messaging, but for now, we should be consistent.